### PR TITLE
Responsiveness

### PR DESCRIPTION
**Pull request by KatarzynaZaloba**
I think it's solved, but in the meantime I changed the list and it can't be visible as tile near tile.. Can you check by your smart eye where I made a mistake?

image

### CODE REVIEW

File: src/feature/MovieDetails/MovieDeatilsPage/Cast/styled.js [hunk 1]
```diff
@@ -2,75 +2,93 @@ import styled from "styled-components";
...
-}
-@media(max-width: ${({ theme }) => theme.breakpoints.mobileMax}px) {
-    grid-template-columns: repeat(3, 1fr);
-}
-@media(max-width: ${({ theme }) => theme.breakpoints.mobileMin}px) {
-    grid-template-columns: repeat(2, 1fr);
-    margin: 12px auto 0;
-}
+    margin: 52px auto 40px;
+    padding: 0 16px;
```
Comment (AdrianJar) [inline src/feature/MovieDetails/MovieDeatilsPage/Cast/styled.js:9]: This padding, I don't think we need it, when you remove it it makes 4 tiles to render in row not 3. Is this padding important?

File: src/feature/MovieDetails/MovieDeatilsPage/MovieTile/styled.js [hunk 2]
```diff
@@ -3,73 +3,79 @@ import { ReactComponent as StarIcon } from "../image/star.svg";
 
 export const ContainerTile = styled.div`
  max-width: 1368px;
- margin: auto;
+ max-height: 100%;
+ margin: 52px auto 40px;
+ padding: 0 16px;
```
Comment (AdrianJar) [inline src/feature/MovieDetails/MovieDeatilsPage/MovieTile/styled.js]: Same here

**Comment by AdrianJar** [comment]
Zrzut ekranu 2023-05-10 o 01 58 44
Grid needs to be moved to wrapper this fixed problem with cast and crew :D

**Review by AdrianJar — APPROVED** [review]
We just need to figure out what to do with padding

**Comment by KatarzynaZaloba** [comment]
We managed another task together! 
image

**Review by Marianna-Weychan — APPROVED** [review]
congratulations Kasia and Adrian! Great job, that's what a group project is all about, to solve problems together :)

**Pull request merged**